src/test/java/api/support/RestAssuredClient.java

```diff
@@ -174,4 +174,8 @@ public Response delete(URL location, int expectedStatusCode, String requestId) {
       .statusCode(expectedStatusCode)
       .extract().response());
   }
+
+  public static RestAssuredClient defaultRestAssuredClient() {
+    return new RestAssuredClient(getOkapiHeadersFromContext());
+  }
```

Comment:
Is this a factory method? If so, please could it be moved to the top of the class for easier discovery.

Reply:
Moved at beginning of the class file